common/rpc/grpc.go

```diff
@@ -123,10 +121,7 @@ func Dial(hostName string, tlsConfig *tls.Config, logger log.Logger, interceptor
 		grpc.WithConnectParams(cp),
 	}
 
-	return grpc.Dial(
-		hostName,
-		dialOptions...,
-	)
+	return grpc.NewClient(hostName, dialOptions...)
```

Comment:
It looks like there are a bunch of calls to grpc.DialContext that should be switched also?

Reply:
DialContext

Reply:
can we do the ones in service/frontend/fx_test.go and common/telemetry/config.go too?

Reply:
replaced both.